src/elements/Label/Label.js

```diff
@@ -106,7 +105,7 @@ Label.propTypes = {
   /** Additional text with less emphasis. */
   detail: PropTypes.string,
 
-  /** Same as detail but formatted as a link. */
+  /** Format the detail as a link. */
   detailLink: PropTypes.string,
```

Comment:
Updated this behavior to match the `link` behavior.  Both just format the component as a link.